File: dayTwo/python/afterBreak/question_one.py
def age_calc( current_fathers_age, current_sons_age):
    number_of_years = 0
    if(current_fathers_age/2 > current_sons_age):
        number_of_years = current_fathers_age - (current_sons_age *  2)
            
        print("The father was twice his son's age ", number_of_years, "years ago")
            
            
    else:
        number_of_years =   (current_sons_age*2) - current_fathers_age
        print("The father will be  twice his son's age in ", number_of_years, " years time")





fathers_age = 38
sons_age = 18
age_calc(fathers_age, sons_age)
   
    

File: dayTwo/java/afterBreak/QuestionOne.java


public class QuestionOne{
    public static void main (String[] args){
    
    int fathersAge = 38;
    int sonsAge = 18;
    ageCalc(fathersAge, sonsAge);
    
    }
    
    
    public static void ageCalc(int currentFathersAge, int currentSonsAge){
        int numberOfYears = 0;
        if(currentFathersAge/2 > currentSonsAge ) {
            numberOfYears = currentFathersAge - (currentSonsAge*  2);
            
            System.out.print("The father was twice his son's age " + numberOfYears + "years ago");
            }
            
        else{
          numberOfYears =   (currentSonsAge*2) - currentFathersAge;
          System.out.print("The father will be  twice his son's age in " + numberOfYears + " years time");
        }     
    }

}
